### DISCUSSION TITLE
Beitrag bearbeiten: Text Modul: Tastatur-Unterstützung verbessern

### DISCUSSION THREAD

**Discussion by techotline** [lotta-schule/web#245] [Ideen]
### Bereich

Etwas ganz anderes

### Verhalten

Verhalten

### Problem

Es ist nicht aktuell noch nicht möglich, nützliche Shortcuts (wie z.B. die 'Eingabe widerrufen' mit Com+Z) in einem Textmodul anzuwenden. Ziel sollte sein, außer Com+C; V; B; I; U das Angebot der Kombinationen, die aktuell in Lotta möglich sind, zu erweitern.

### Beschreibung

Erwartungen an den Editor sind:

Cmd+Z, Cmd+Sh+Z (undo, redo)
"-" für neuen Listeneintrag
+ evtl. weitere Vorschläge?

**Comment by ptitmouton** [comment]
Da hier eine offene Frage gestellt wird habe ich das mal als Diskussion angelegt, da können wir ein Ticket draus machen wenn sich alle einig sind.

Tastatur finde ich grundsätzlich immer gut, zumal ich ja dieses Ticket hier ja schon selbst mal vor 3 Jahren nach einer frustrierenden Erfahrung angelegt hatte (bei gitlab ... https://gitlab.com/lotta-schule/web-client/-/issues/641).

Problem ist: Ich fass slate immer nur mit der Kneifzange an, weshalb wir auch eine recht veraltete Version fahren, bei der es mir jetzt schon graut die zu aktualisieren.
Danach müsste History-Support überhaupt erst eingebaut werden, das können wir ja noch gar nicht.
Dann auf die Tastatureingabe selbst zu reagieren  ist der geringste Schritt.

Ich unterstütze diesen Vorschlag und warte auf sinnvolle Ergänzungen. Bin mir sicher @BillyTheKing  und @wie0815 benutzen "klassische" Textverarbeitung deutlich öfter und können hier mehr Input als ich geben, ich glaube meine Shortcuts wollt ihr nicht nutzen ...

**Comment by BillyTheKing** [comment]
Funktion können wir auf dem Schirm haben, ist aber meiner Ansicht nach keines unserer dringendsten "Baustellen".
